packages/api/.eslintrc.json

```diff
@@ -0,0 +1,15 @@
+{
+    "env": {
+        "node": true,
+        "commonjs": true,
+        "es2020": true
+    },
+    "extends": [
+        "airbnb-base"
+    ],
+    "parserOptions": {
+        "ecmaVersion": 11
+    },
```

Comment:
I think this is redundant with `"es2020": true` from above, which sets this `ecmaVersion` [automatically](https://eslint.org/docs/user-guide/configuring#specifying-environments). Is being explicit recommended/particularly desirable here?